packages/components/README.md

```diff
@@ -1,104 +1,7 @@
-
-- [Button](src/components/Button/README.md)
-- [Heading](src/components/Heading/README.md)
-- [Image](src/components/Image/README.md)
-- [RichText](src/components/RichText/README.md)
-- [Text](src/components/Text/README.md)
-
-## Getting started
-
-### Installation
-
-```bash
-npm install @contentful/experiences-components-react
-```
-
-## Styling
-
-By default, the components are unstyled. This allows you to style the components to match your brand and design system. If you want a set of default styles to get started, see below.
-
-### Adding custom styles
-
-Each component has a css class that you can use to add your own styles. The classes are named in the style of `cf-{component-name}` (ie `cf-button`).
-
-For example, to style the `Button` component, you can do the following:
-
-```css
-.cf-button {
-  /* your styles here */
-}
-```
-
-All components also support passing in custom class names via the `className` prop. This allows you to add your own class names to the component, which you can then use to style the component.
-
-## withExperienceBuilder util
-
-We provide a helper function (as a [higher-order-component](https://legacy.reactjs.org/docs/higher-order-components.html)) to make it easier to register your own custom components with Experience Builder. This function helps ensure your component has all the required props and is properly registered with Experience Builder.
-
-### Usage
-
-```jsx
-import { withExperienceBuilder } from '@/utils/withExperienceBuilder';
-import { MyComponent } from './MyComponent';
-
-export const ExperienceBuilderMyComponent = withExperienceBuilder(
-  // Your component
-  MyComponent,
-  // component registration configuration for EB
-  {
-    id: 'my-component',
-    name: 'My Component',
-    category: 'Custom',
-    variables: {
-      label: {
-        type: 'Text',
-        defaultValue: 'My Component',
-      },
-    },
-  },
-);
-```
-
-### Container wrapping
-
-By default, the `withExperienceBuilder` function will not wrap your component in a container. However, it is often useful to have your component wrapped. If the components is wrapped, all the styles generated from Experience Builder will be applied to the wrapping container instead of the component itself. This will make it so the additional styles don't interfere with your component's styles.
-
-To wrap your component, pass in the `wrapComponent` option:
-
-```jsx
-export const ExperienceBuilderMyComponent = withExperienceBuilder(
-  // Your component
-  MyComponent,
-  // component registration configuration for EB
-  { /* EB config */ },
-  // wrap the component with a container (defaults to false)
-  { wrapComponent: true }
-);
-```
-
-You can also provide the tag name the container will use (which defaults to 'div'):
-
-```tsx
-{ wrapComponent: true, wrapContainerTag: 'span' }
-```
\ No newline at end of file
+Please refer to our [Documentation](https://www.contentful.com/developers/docs/experiences/) to learn more about it.
```

Comment:
Shouldn't we point to a more specific and relevant doc here? Maybe https://www.contentful.com/developers/docs/experiences/register-custom-components/#built-in-components

Reply:
Thanks for the feedback!  I agree and updated the link